src/app/process-editor/services/bpmn-js/element.helper.ts

```diff
@@ -35,16 +35,12 @@ export const ElementHelper = {
         }
     },
 
-    setProperty(modeling: Bpmn.Modeling, element: Bpmn.DiagramElement, propertyName: BpmnProperty, value: any): void {
-        try {
+    setProperty(modeling: Bpmn.Modeling, element: Bpmn.DiagramElement, propertyName: BpmnProperty, value: any, moddle: Bpmn.Moddle): void {
             const handler = getHandler(propertyName);
             const set = handler.set;
 
-            set(modeling, element, value);
-        } catch (error) {
-            /*tslint:disable-next-line*/
```

Comment:
I'd rather put this try/catch back, with logging a developer friendly error message, and rethrowing the error if needed?!